libphobos/m4/druntime/libraries.m4

```diff
@@ -16,6 +40,17 @@ AC_ARG_ENABLE(thread-lib,
 ])
 
 
+# DRUNTIME_LIBRARIES_TIME
+# -----------------------
++# Check whether -lrt is required for sched_yield, nanosleep, or clock_gettime.
+AC_DEFUN([DRUNTIME_LIBRARIES_TIME],
+[
+AC_SEARCH_LIBS([sched_yield], [rt])
+AC_SEARCH_LIBS([nanosleep], [rt])
+AC_SEARCH_LIBS([clock_gettime], [rt])
+])
+
```

Comment:
Attempt number 1 - horrible implementation ahead.  https://github.com/ibuclaw/GDC/commit/285c742630b3e528a69b463eaa914d610689bb31

It writes the following bytecode to assembler `"%.*s%.*s", sizeof(len), &len, len, str` to the section `.d_pragma_libs`.  So for example:
```
pragma(lib, "dl");
pragma(lib, "curl");
```
Will have the data `2dl4curl` written to the section.

Next step is to parse that out, and append `-ldl` and `-lcurl` to the drivers `outfiles`, the rest is taken care of by the driver.